okhttp/src/main/java/okhttp3/CertificatePinner.kt

```diff
@@ -264,21 +242,6 @@ data class CertificatePinner internal constructor(
       return hostname == canonicalHostname
     }
 
-    override fun equals(other: Any?): Boolean {
-      return other is Pin
-          && pattern == other.pattern
-          && hashAlgorithm == other.hashAlgorithm
-          && hash == other.hash
-    }
-
-    override fun hashCode(): Int {
```

Comment:
Did you mean to make this a data class? Or is it just not compared or used as a key?

Reply:
I meant to make it a data class.

Reply:
Good catch!